HISTORY.rst

```diff
@@ -2,6 +2,22 @@
 History
 =======
 
+0.15.2 (2023-XX-XX)
+-------------------
+Contributor: [Fernando Aristizabal](https://github.com/fernando-aristizabal)
```

Comment:
Please fix the link following the RST format that I mentioned in the previous comment.